daemon/config/config.go

```diff
@@ -110,3 +115,96 @@ func (cfg *Config) Validate() error {
 
 	return nil
 }
+
+//MergeConfigConfigurations merges flagSet flags and config file flags into Config.
+func (cfg *Config) MergeConfigConfigurations(config *Config, flagSet *pflag.FlagSet) error {
```

Comment:
merge config configuration, do we have duplicated config in this function naming?

Reply:
how about MergeConfiguration.

Reply:
yeah, that is much better.